kitsune/forums/feeds.py

```diff
@@ -12,7 +13,10 @@ class ThreadsFeed(Feed):
     feed_type = Atom1Feed
 
     def get_object(self, request, forum_slug):
-        return get_object_or_404(Forum, slug=forum_slug)
+        forum = get_object_or_404(Forum, slug=forum_slug)
+        if not forum.allows_viewing_by(request.user):
```

Comment:
I am wondering whether it's a better idea to add a method in the `Forum` class similar to `authorized_forums_for_user` for consistency. In any case this is a nit

Reply:
Hm, I don't see how that would work here, but maybe I'm misunderstanding something? It looks like the `authorized_forums_for_user` class method is not used anywhere.

Reply:
It could be used but I am starting to second guess the efficiency of that method. Pls ignore